content/en/examples/pods/pod-with-toleration.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    disktype: ssd
```

If keeping a `nodeSelector`, how about:
```suggestion
    kubernetes.io/os=linux
````